### ISSUE TITLE
Tooltips are not announced by screen readers

### ISSUE THREAD

**Issue by hippee-lee** [clr-team/planning#47]
## Description of work

Tooltips are not announced when you tab and focus on them. 
ClarityUI tooltips and Clarity Angular tooltips need to announce and , after discussions with @chlane, keep the tooltip open when content has hover state. 

## Issue(s)

* https://github.com/vmware/clarity/issues/3525

## Reviewers

* Bozhidar
* Colin
* Cory
* Ivan
* Jeeyun
* Jeremy
* Matt
* Scott
* Shijir

## Definition of done

* [ ] Tests completed
* [ ] Reviewed by peers
* [ ] Design sign off (if necessary)
* [ ] Merged into master/topic branch
* [ ] Backport (if necessary)

**Comment by hippee-lee** [comment]
This is a duplicate of #42

**Comment by hippee-lee** [comment]
I put this (duplicate) into done column so it doesn't affect the points.
